dapp/src/components/shared/InfoTooltip.tsx

```diff
@@ -1,11 +1,20 @@
 import React from "react"
-import { Info } from "#/assets/icons"
+import { IconInfoCircleFilled } from "@tabler/icons-react"
 import { Icon, Tooltip, TooltipProps } from "@chakra-ui/react"
 
+// TODO: Define in the new color palette
+const ICON_COLOR = "#3A3328"
```

Comment:
Note:
I did the same [here](https://github.com/thesis/acre/pull/839). Let's stick to this before we refactor Chakra theme.